app/src/forms/submission/service.js

```diff
@@ -10,6 +9,7 @@ const {
   SubmissionAudit,
   SubmissionMetadata
 } = require('../common/models');
+const {reArrangeFormSubmissionJSon} = require('../common/utils');
```

Comment:
The imported value should have spacing around its name to keep things consistent with everything else. const { ... } instead of {...}